boost-common/src/main/java/boost/common/boosters/AbstractBoosterConfig.java

```diff
@@ -46,10 +46,14 @@
     public static final String MP_CONFIG_VERSION_13 = "1.3-" + boostVersion + "-" + milestone + "-SNAPSHOT";
```

Comment:
Can we change "boostVersion to "1.0" (comment left over from PR #328) ?

Reply:
done